.changeset/sweet-peaches-invite.md

```diff
@@ -0,0 +1,5 @@
+---
+'@chainlink/ion.au-adapter': minor
+---
+
+added PR notes
```

Comment:
The one changeset with the major version should be enough for this PR. These would just get wrapped up under it anyways so you can remove these minor ones.